pkg/roachprod/install/cluster_synced.go

```diff
@@ -1712,7 +1712,7 @@ VERSION=$(%[1]s version --build-tag)
 VERSION=${VERSION::5}
 TENANT_SCOPE_OPT=""
 if [[ $VERSION = v22.2 ]]; then
-       TENANT_SCOPE_OPT="--tenant-scope 1,2,3,4,11,12,13,14"
+       TENANT_SCOPE_OPT="--tenant-scope $(echo {1..100} | tr ' ' ,)"
```

Comment:
Nit: I'd still enclose the delimiter in quotes for ease of (human) parsing; i.e., `','`